### PR TITLE
state personnel - small entries

### PR DESCRIPTION
**Pull request by brendansudol**
very similar to what was done with expenses

preview:
image

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author

### This feature is done when...
- [ ] Design has approved the experience

### CODE REVIEW

File: web/src/util/formats.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { format } from 'd3-format';
 
-const isNumeric = n => !Number.isNaN(parseFloat(n)) && Number.isFinite(n);
-
 const BIG_NUMBER_THRESHOLD = 1000000;
 
+// eslint-disable-next-line no-restricted-globals
+export const isNumeric = n => !isNaN(parseFloat(n)) && isFinite(n);
```
Comment (mgwalker) [inline web/src/util/formats.js]: Why the change from `Number.isNaN` to the older version?

Comment (brendansudol) [reply web/src/util/formats.js]: our old fn was saying that `'0.00'` was not numeric -- this one solves that, though now I see that it's due to the `isFinite` change...

image

**Comment by brendansudol** [comment]
sure thing @lauraponce

File: web/src/util/formats.js (hunk 1 above)
Comment (mgwalker) [reply web/src/util/formats.js]: What if we simplified to:

```js
export const isNumeric = n => Number.isFinite(Number.parseFloat(n));
```

It turns out that `NaN` is not considered finite, the `isFinite` check should be enough, and then we get to use the scoped methods instead of the global ones.  Since this PR is already merged, we can put this on the backburner if it makes sense and change it sometime in the future when we happen to be fiddling in the utils file again.
